core/dbt/task/run.py

```diff
@@ -483,3 +484,30 @@ def get_runner_type(self, _):
     def task_end_messages(self, results):
         if results:
             print_run_end_messages(results)
+
+    def manage_schema(self, adapter, results: List[RunResult]):
+        # adapter == postgressadapter
+        compile_nodes: List[ParsedSourceDefinition] = list(filter(lambda r: type(r.node) == ParsedSourceDefinition, results))
+        for r in results:
+            print(dir(r))
+            print(type(r))
+        database_schema_set: Set[Tuple[Optional[str], str]] = {
+            (r.node.database, r.node.schema, r.node.table)
+            for r in results
+            if r.node.is_relational
+            and r.status not in (NodeStatus.Error, NodeStatus.Fail, NodeStatus.Skipped)
+        }
+        print("DB", database_schema_set)
+        print("type of graph", self.graph.nodes().values())
+        # Find all models with manage_schema config
+        #   {%- for node in graph.nodes.values() | selectattr("resource_type", "equalto", "model") | list
+        # Collect "schema" next to manage_schema config
+        # Find a list of relations in a given schema
+        # List everything in the schema
+        # Drop the difference
+        print("TYPE IS", type(adapter))
```

Comment:
I really need a nice working debug workflow or isolated test to get this working properly :D